libra_toolbox/neutron_detection/activation_foils/calibration.py

```diff
@@ -26,6 +26,7 @@ class Nuclide:
     intensity: List[float] = None
     half_life: float = None
     atomic_mass: float = None
+    abundance: float = None
```

Comment:
Because it's used in a calculation, I think this should default to `1` and not `None`

```suggestion
    abundance: float = 1.0
```